modules/nw-sriov-supported-devices.adoc

```diff
@@ -14,5 +14,6 @@
 * Mellanox MT27800 Family [ConnectX-5] 100GbE with vendor ID `0x15b3` and device ID `0x1017`
 * Mellanox MT27700 Family [ConnectX-4] VPI adapter card, EDR IB (100Gb/s), single-port QSFP28 with vendor ID `0x15b3` and device ID `0x1013`
 * Mellanox MT27800 Family [ConnectX-5] VPI adapter card, EDR IB (100Gb/s), single-port QSFP28 with vendor ID `0x15b3` and device ID `0x1017`
+* Mellanox MT28880 Family [ConnectX-5 Ex] VPI adapter card, EDR IB (100Gb/s), dual-port QSPF28 with vendor ID `0x15b3` and device ID `0x1019`
```

Comment:
I don't have a CX-5 Ex card so I cannot check, but from the [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1937884) request, it is a Ethernet controller, not VPI:

5e:00.0 Ethernet controller: Mellanox Technologies MT28800 Family [ConnectX-5 Ex]

I'll double check with FDP to see what exact model has been verified.
What I knew is FDP only verify Ethernet functions, not InfiniBand.

Reply:
@mikemckiernan  I just confirmed with @ctrautma that we only verified CX-5 Ex Ethernet Controller. please remove the "VPI adapter card".